cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/diff/change/parsing/MapChangeParsingStrategy.java

```diff
@@ -41,18 +50,28 @@
         return Arrays.asList(MapChange.class);
     }
 
-    private CfMapValueChanged parseMapEntry(EntryChange entryChange) {
+    private CfMapValueChanged parseMapEntry(String propertyName, EntryChange entryChange) {
         if (entryChange instanceof EntryAdded) {
+            log.verbose("Appending", propertyName,"map change with added entry:"
+                    , entryChange.getKey() + ":", ((EntryAdded) entryChange).getValue());
```

Comment:
This formatting is a bit weird. Consider putting the comma in the line above.

Reply:
Would also recommend to use `debug` logging here